src/main/java/mesosphere/marathon/client/model/v2/App.java

```diff
@@ -9,25 +9,26 @@
 import mesosphere.marathon.client.utils.ModelUtils;
 
 public class App {
-	private String id;
-	private String cmd;
-	private String version;
-	private Integer instances;
-	private Double cpus;
-	private Double mem;
-	private Collection<String> uris;
-	private List<List<String>> constraints;
-	private Container container;
-	private Map<String, String> env;
-	private Map<String, String> labels;
-	private String executor;
-	private List<Integer> ports;
-	private Collection<Task> tasks;
-	private Integer tasksStaged;
-	private Integer tasksRunning;
+    private String id;
```

Comment:
- [ ] Here and throughout, revert these whitespace changes.  
They add noise to this change and make it harder to evaluate what's materially different.